lib/re.mli

```diff
@@ -371,6 +371,7 @@ val pp_re : Format.formatter -> re -> unit
 (** Alias for {!pp_re}. Deprecated *)
 val print_re : Format.formatter -> re -> unit
 
+val witness : t -> string
```

Comment:
While I do need it for tyre, the functionality itself is not tyre-specific (and, as I said, well specified).